src/ZString/Utf16ValueStringBuilder.cs

```diff
@@ -119,7 +119,6 @@ public void Clear()
 
         public void TryGrow(int sizeHint)
         {
-
```

Comment:
Noise in the delta. Unsolicited code formatting change - empty lines must be preserved.